packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts

```diff
@@ -160,3 +166,21 @@ export const createAnnouncementString = (
     numOfParticipants: numberOfExtraParticipants.toString()
   });
 };
+
+/* @conditional-compile-remove(dtmf-dialer) */
+/**
+ * determines if the media gallery should be replaced by the dtmf dialer
+ * @param callees Target callees to determine if the dtmf dialer should be shown
+ * @returns whether the dialer should be the gallery content or not
+ */
+export const showDtmfDialer = (callees?: CommunicationIdentifier[]): boolean => {
+  return !!callees?.filter((callee) => {
+    if (isPhoneNumberIdentifier(callee)) {
+      return true;
+    } else if (isMicrosoftTeamsAppIdentifier(callee)) {
+      return true;
+    } else {
+      return false;
```

Comment:
nit; something like this
```suggestion
  return !!callees?.filter((callee) => {
    return isPhoneNumberIdentifier(callee) || isMicrosoftTeamsAppIdentifier(callee))
```